archivesunleashed/auk
# Add Neighbours highlight on mouseover.

**The title of this pull-request should be a brief description of what the pull-request fixes/improves/changes. Ideally 50 characters or less.**

- Add arrows to show direction of links.
- Highlights a node's neighbours on hover. (Greys out non-neighbours).

**GitHub issue(s)**:

If you are responding to an issue, please mention their numbers below.

* ex: #130 


# What does this Pull Request do?

Easiest is with screen shots:

![image](https://user-images.githubusercontent.com/111091/41999063-7095be20-7a2a-11e8-9bc6-b701f41bb475.png)

to:
![image](https://user-images.githubusercontent.com/111091/41999084-815a852e-7a2a-11e8-87c7-bdfa63104e77.png)

When you mouse over "nanaimodailynews"

# How should this be tested?

- Hover-over should work in regular and full-screen.
- It should also work if you back out of a screen and go to another network.
- Should pass Travis and linting tests.
- Arrow sizing and interaction should work according to UI preferences of the team (ie. I can make things bigger, louder or whatnot as desired).

# Additional Notes:

# Interested parties

@ianmilligan1 @SamFritz 

Thanks in advance for your help with the Archives Unleashed Toolkit!
